### ISSUE TITLE
Preapred statement does not exists when deploying to postgres

### ISSUE THREAD

**Issue by dgiddins** [sqitchers/sqitch#717]
I'm getting the following error while trying to deploy changes to our test and production servers.

```
Could not deallocate
[14:43:21 ](https://teamcity.otenv.com/buildConfiguration/LocationsTeam_LocationsV5_LocationReadSchema_2bDeployToPpRs/20979384?buildTab=log&focusLine=111&logView=flowAware&linesState=111)         ERROR: 7 "ERROR:  prepared statement "dbdpg_p1_1" does not exist" (err#0)
```

This might be related to and upgrade to postgres 13, but seems unlikley as this error started happening some time after the upgrade. 

If you need more info on setup, please list what is missing from this post and I will gather what is needed to debug this.

**Comment by theory** [comment]
Are you using pgBouncer or another proxy server? Good way to lose prepared statements.

**Comment by dgiddins** [comment]
I believe we might be, much of the detail of our deployment is hidden away from us. It does sometimes, very infrequently work which isn't helping either. Is there a way to bypass pgbouncer?

**Comment by theory** [comment]
Yeah, if you can connect to the port that Postgres is actually listening on. The other thing to do is to change [this line](https://github.com/sqitchers/sqitch/blob/aace1e837b5e07fa0d8cfb924505b221f58079c4/lib/App/Sqitch/Engine/pg.pm#L129) to pass `0` instead of `1`. There ought to be a way to turn it on or off in the connection string, though…

**Comment by theory** [comment]
Alas, there is no [connection parameter](https://www.postgresql.org/docs/15/libpq-connect.html#LIBPQ-PARAMKEYWORDS) for enabling or disabling server side prepares :-(
